### PR TITLE
Add lazyvim to editors.md

### PR DESCRIPTION
**Pull request by KonnorRogers**
### Motivation

https://x.com/vinistock/status/1799641482980421935

### Implementation

install lazyvim, open a ruby file, call `:LspInfo` and see if ruby_lsp is running

### Automated Tests

just docs

### Manual Tests

install lazyvim, open a ruby file, call `:LspInfo` and see if ruby_lsp is running

### CODE REVIEW

**Comment by joshuap** [comment]
To make this work with asdf, I had to override the `cmd` like this:

```
return {
  {
    "neovim/nvim-lspconfig",
    ---@class PluginLspOpts
    opts = {
      ---@type lspconfig.options
      servers = {
        -- disable solargraph from auto running when you open ruby files
        solargraph = {
          autostart = false,
        },
        -- ruby_lsp will be automatically installed with mason and loaded with lspconfig
        ruby_lsp = {
          cmd = { "/Users/josh/.asdf/shims/ruby-lsp" },
        },
      },
    },
  },
}
```

Related: #1248

**Comment by KonnorRogers** [comment]
> To make this work with asdf, I had to override the `cmd` like this:
> 
> ```
> return {
>   {
>     "neovim/nvim-lspconfig",
>     ---@class PluginLspOpts
>     opts = {
>       ---@type lspconfig.options
>       servers = {
>         -- disable solargraph from auto running when you open ruby files
>         solargraph = {
>           autostart = false,
>         },
>         -- ruby_lsp will be automatically installed with mason and loaded with lspconfig
>         ruby_lsp = {
>           cmd = { "/Users/josh/.asdf/shims/ruby-lsp" },
>         },
>       },
>     },
>   },
> }
> ```
> 
> Related: #1248

Interesting...I use ASDF and it ran without issue 🤔 

I have installed ruby_lsp gem globally before with my global ruby version so perhaps that's why it worked?

File: EDITORS.md [hunk 1]
```diff
@@ -109,6 +110,32 @@ require("lspconfig").ruby_lsp.setup({
...
+    ---@class PluginLspOpts
+    opts = {
+      ---@type lspconfig.options
+      servers = {
+        -- disable solargraph from auto running when you open ruby files
+        solargraph = {
+          autostart = false
+        },
+        -- ruby_lsp will be automatically installed with mason and loaded with lspconfig
+        ruby_ls = {},
```
[the remark this answers is not in the document]
Comment (KonnorRogers) [reply EDITORS.md]: Oops! Good catch!

**Comment by vinistock** [comment]
Thank you for the contribution!

**Pull request merged**